include/quiche.h

```diff
@@ -314,6 +314,13 @@ void quiche_conn_on_timeout(quiche_conn *conn);
 int quiche_conn_close(quiche_conn *conn, bool app, uint64_t err,
                       const uint8_t *reason, size_t reason_len);
 
+// Configures the given session for resumption.
+int quiche_conn_set_session(quiche_conn *conn, const uint8_t *buf, size_t buf_len);
+
+// Returns the serialized cryptographic session for the connection..
+void quiche_conn_get_session(quiche_conn *conn, const uint8_t **out,
```

Comment:
Should follow the naming of the Rust function here IMO, i.e. `quiche_conn_session()`.

Reply:
Nit: Also, we try to order functions here like in `lib.rs`, so both of these should be moved around (same applies `ffi.rs`).

Reply:
> Should follow the naming of the Rust function here IMO, i.e. quiche_conn_session().
No strong preference here... I just thought as we have a `set` version adding `get` makes it more clear what this is all about.